host/xdc/integration_failover_test.go

```diff
@@ -158,10 +158,8 @@ func (s *integrationClustersTestSuite) TestDomainFailover() {
 
 	// update domain to fail over
 	updateReq := &types.UpdateDomainRequest{
-		Name: common.StringPtr(domainName),
-		ReplicationConfiguration: &types.DomainReplicationConfiguration{
-			ActiveClusterName: common.StringPtr(clusterName[1]),
-		},
+		Name:              domainName,
+		ActiveClusterName: common.StringPtr(clusterName[1]),
```

Comment:
Since there's just one, this makes sense IMO, but I am kinda curious why it changed.

Reply:
This is was actually fixed with #3967 and no longer there after rebase.